jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/uper/UperUtil.java

```diff
@@ -123,6 +130,28 @@ public static String determineMessageType(OdeMsgPayload payload) {
 		return messageType;
 	}
 
+    /**
+		* Trims extra `00` bytes off of the end of an ASN1 payload string
```

Comment:
Are the trailing 0's just padded bytes? Should we include that in here so we know _why_ we're removing them?

Reply:
Yeah, they are extra 0's that we add as part of the ingestion process when we create the buffer. I'll add a comment on this.